src/data/accessibility-quiz.ts

```diff
@@ -1,4 +1,13 @@
 const accessibilityQuiz = [
+  {
+    Question: "What legal repercussions can businesses face for failing to meet website accessibility requirements?",
+    Answer: "Businesses that fail to meet website accessibility requirements may face lawsuits, financial penalties, and reputational damage, in addition to the potential loss of customers and legal compliance obligations.",
+    Distractor1: "What are the top five e-commerce trends for 2023?",
+    Distractor2: "How can businesses optimize their supply chain management processes?",
+    Distractor3: "What are the benefits of adopting cloud computing in the corporate sector?",
```

Comment:
we shouldn't use questions for the distractors.

```suggestion
    Distractor1: "no repercussions",
    Distractor2: "they will serve 10 years in jail",
    Distractor3: "they will do 2500 hours of community service",
```